src/main/java/seedu/address/logic/commands/AddStudentCommand.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_STUDENTS;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.student.Id;
+import seedu.address.model.student.Student;
+import seedu.address.model.student.Training;
+
+/**
+ * Edits the details of an existing student in the address book.
+ */
+public class AddStudentCommand extends Command {
```

Comment:
Will this class name be confusing since it sounds like adding student to the student list also (i.e. `AddCommand`)? Just wondering only

Reply:
hmm I wanted to put AddStudentToTraining but that would be bit too verbose I think, if others are okay with it ill refactor

Reply:
think might be clearer for AddStudentToTrainingCommand since AddStudentCommand can be confused with adding a student